app/src/main/java/com/example/pix/chat/fragments/ChatFragment.java

```diff
@@ -143,11 +150,24 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
             Toast.makeText(getContext(), "Error retrieving more chats", Toast.LENGTH_SHORT).show();
         }
 
-        ivProfile.setOnClickListener(view12 -> getParentFragmentManager()
-                .beginTransaction()
-                .addToBackStack("stack")
-                .replace(R.id.friend_container, new ProfileFragment(friend))
-                .commit());
+        Fragment friendProfile = new ProfileFragment(friend);
+        ivProfile.setOnClickListener(view12 -> {
```

Comment:
I am planning to fix these small "view12" and/or missing constants after I merge.